### DISCUSSION TITLE
I'd like to be able to filter a PR search by the number of reviews that the PRs have

### DISCUSSION THREAD

**Discussion by calvinballing** [community/community#38061] [Pull Requests]
For example, to find PRs with fewer than 2 reviews

**Comment by david-engelmann** [comment]
Would adding "most reviewed" and "least reviewed" to the sort section be sufficient?

**Reply by calvinballing** [reply]
That would would probably do it, especially if the `Approved` note here were updated to show the number of approvals such as `1 Approval`, `2 Approvals`
<img width="421" alt="image" src="https://user-images.githubusercontent.com/5534029/199497745-bd494372-6e21-42fa-b116-8a3f338d2e12.png">

**Reply by Lasercar** [reply]
I was just about to start a new discussion for this, though it looks like there's already one (but it's old).

I'd really like to be able to sort PRs by the most/least approved reviews.

That, and it'd be nice to search PRs that have been reviewed by multiple specific people.
